README.md

```diff
@@ -6,17 +6,14 @@
 
 
 # Introduction 
-keras2onnx enables you convert the keras models into [ONNX](https://onnx.ai).
-Initially Keras converter was developer in the project onnxmltools. To support more kinds of keras models and reduce the complexity of mixing multiple converters, keras2onnx was created to convert the keras model only.
+This Keras model converter(keras2onnx) enables you convert the keras models into [ONNX](https://onnx.ai) model format .
+Initially Keras converter was developed in the project onnxmltools. To support more kinds of keras models and reduce the complexity of mixing multiple converters, keras2onnx was created to convert the keras model only.
 
-keras2onnx supports the keras lambda/custom layer by parsing the TF graph built from Keras model.
+keras2onnx converter supports not only all keras built-in layers, but also the lambda/custom layer by working with [tf2onnx](https://github.com/onnx/tensorflow-onnx) converter which was embedded into the source tree directly to avoid the version conflict and install complexity. 
```

Comment:
... which _is_ embedded into ...
... to avoid _version conflicts and installation_ complexity.